docs/installation/server-config/security.rst

```diff
@@ -76,6 +76,14 @@ KSQL to Schema Registry SSL communication.
 
 The exact settings will vary depending on the encryption and authentication mechanisms the
 Confluent Schema Registry is using, and how your SSL certificates are signed.
+
+You can pass authentication settings to the Schema Registry client used by KSQL
+by adding the following to your KSQL server config.
+
+.. code:: bash
+    ksql.schema.registry.basic.auth.credentials.source=USER_INFO
+    ksql.schema.registry.basic.auth.user.info=username:password
```

Comment:
for user.info you actually have to specify `ksql.schema.registry.schema.registry.basic.auth.user.info`, because of the way the schema-registry configs are currrently named.